lib/workers/repository/index.js

```diff
@@ -14,6 +15,23 @@ module.exports = {
   renovateRepository,
 };
 
+async function setGitPrivateKey(config) {
+  if (config.gitPrivateKey && config.localDir) {
+    try {
+      const cwd = config.localDir;
+      const git = Git(cwd).silent(true);
+      await git.raw([
+        'config',
+        '--global',
```

Comment:
Not global. We want this git config setting to be run locally so it doesn't affect the user's own git settings

Reply:
Got it :+1: